public/pages/CreateMonitor/components/HTTPInput/URLInfo.js

```diff
@@ -0,0 +1,159 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React, { Fragment } from 'react';
+import {
+  FormikFieldText,
+  FormikSelect,
+  FormikFieldNumber,
+  FormikFieldRadio,
+} from '../../../../components/FormControls';
+import 'brace/mode/json';
+import 'brace/mode/plain_text';
+import 'brace/snippets/javascript';
+import 'brace/ext/language_tools';
+import { hasError, isInvalid } from '../../../../utils/validate';
+import { EuiSpacer } from '@elastic/eui';
+import { URL_TYPE } from '../../../Destinations/containers/CreateDestination/utils/constants';
+
+const protocolOptions = [{ value: 'https', text: 'HTTPS' }, { value: 'http', text: 'HTTP' }];
+
+const URLInfo = ({ values }) => {
```

Comment:
Copy and paste from URLInfo in Destinations, can we not reuse instead of duplicating?

Reply:
I think we could, but I'm not sure how to change the validation logic to allow`localhost` .